translations/en-us.yaml

```diff
@@ -8569,6 +8569,7 @@ modalAddCloudKey:
   saving:
     validating: Validating Keys
   errors:
+    errorMessage: 'There was a problem validating your credentials. Please try again.'
```

Comment:
perhaps this should be more descriptive - 'authValidationError'?

Reply:
Restructured code to use existing error display setup so this is actually no longer needed.